lib/assets.js

```diff
@@ -12,14 +12,14 @@ var defaults = {
   }
 };
 
-module.exports = function (gulp, options) {
-  var opts = merge({}, defaults, options);
-  var dest = opts.dest;
-  var optimize = opts.optimize;
-  var plugins = opts.plugins;
-  var src = opts.src;
+module.exports = (gulp, options) => {
+  const opts = merge({}, defaults, options);
```

Comment:
`const` feels like an awkward (though technically acceptable) choice for function scope variables. It implies that no value changing or reassignment will occur, and while this appears to be the case for these functions, I think it's pretty common for something like this occur within a function body:

```js
function foo () {
  var result = false; // a default value
  if (condition) result = true; // a new value based on some logic
  return result;
}
```

...which would not work with `const`. IMO, sticking with `var` is preferable here (and in all of these other files.)

Using it for the above `defaults` object seems fine.